_posts/2017-02-14-whats-in-7.0.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "What's in 7.0"
+author: Henry Zhu
+date:   2017-02-14 12:00:00
+categories: announcements
+share_text: "What's in 7.0"
+---
+
+If you didn't know already, we're planning on releasing a 7.0 version soon! I'd like to go over some notable changes and a few things we are still working out!
+
+> For more specific changes, please check out our upgrade guide (will be updated)!
+
+## 💥 Breaking Changes
+
+### [#4315](https://github.com/babel/babel/issues/4315) Drop support for unmaintained Node versions: 0.10, 0.12, 5
+
+When OSS maintainers want to move the project forward and make breaking changes, it may come at the cost of more hassle for their users.
+
+A major version bump *is* just a number, but it can cause unnecessary pain for users/library authors (including ourselves since we'd want to upgrade Babel itself to run on Babel 7 at some point right). We didn't want to do it just for this change.
+
+But by dropping Node 0.12, we can use tools that support a minimum of Node 4 as well such as ESLint, Yarn, and Jest (+Lerna). And other dependencies that have also dropped can be available for us to use as well.
+
+### [#5128](https://github.com/babel/babel/issues/5128) Babel presets must export a function as the default export
+
+We want to standardize on requiring presets to be functions that return a Babel config object (rather than just the object).
```

Comment:
"We want to" seems like we're considering it... what about:

> To simplify things, we've decided to standardize on requiring presets be functions that return a Babel config object (rather than just an object).